blender/arm/props_ui.py

```diff
@@ -43,6 +44,94 @@
     arm.enable_reload(__name__)
 
 
+import bpy
+class ArmUIPropertyGroup(bpy.types.PropertyGroup):
+    show_editor_time_warnings: bpy.props.BoolProperty(
+           name="Show editor time warnings",
+           description="Template",
+           default=True)
+
+
+def label_multiline(text: str, context: bpy.types.Context, parent: bpy.types.UILayout, bullet: bool = False, bulletSymbol: str = "*"):
+    "Same as UILayout.label except with line splitting. If bullet is True we will put an asterix * before the first line and indent the rest."
+    text = str(text)
+    textListSplitByNewLines = text.split('\n')
+    import textwrap
+    for text in textListSplitByNewLines:
+
+        chars = int(context.region.width / 7)   # 7 pix on 1 character
+        amountOfSpaceNecessary = "  "
+        i = -1
+        while bulletSymbol.__len__() > i:
+            i += 1
+            amountOfSpaceNecessary += " "
+        if bullet:
+            wrapper = textwrap.TextWrapper(width=chars, initial_indent=f" {bulletSymbol} ", subsequent_indent=amountOfSpaceNecessary)
+        else:
+            wrapper = textwrap.TextWrapper(width=chars)
+        text_lines = wrapper.wrap(text=text)
+        for text_line in text_lines:
+            parent.label(text=text_line)
+
+
+from enum import Enum
+class WarningSeverityLevels(Enum): # Unfortunately this cant go inside EditorTimeWarnings as Python wont then let me reference it for default values in function parameters within EditorTimeWarnings.
```

Comment:
I haven't tested this, but the enum looks like it could introduce a similar issue than what was fixed by https://github.com/armory3d/armory/pull/2392.

Reply:
The downside of intEnum is that I dont think it can be converted to strings - unlike regular enums.

I will switch the class to use string variables instead for now.

Reply:
Using IntEnum is just one possible solution, the other would be to only define the enum once (don't redefine it when reloading) by moving the declaration into the else branch of the `if arm.is_reload(__name__):` block in the file header. Using strings is also fine for me but you loose a bit of type safety in return.